cmd/logs.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"io"
+	"os"
+	"sync"
+
+	"github.com/apex/log"
+	"github.com/heroku/color"
+)
+
+const (
+	errorLevelText = "ERROR: "
+	warnLevelText  = "Warning: "
+)
+
+// Default logger
+var (
+	Logger = &log.Logger{
+		Handler: &handler{
+			writer: os.Stdout,
+		},
+	}
+	warnStyle  = color.New(color.FgYellow, color.Bold).SprintfFunc()
+	errorStyle = color.New(color.FgRed, color.Bold).SprintfFunc()
+)
+
+func SetLogLevel(level string) {
```

Comment:
why not `func (l *Logger) SetLogLevel(level string)`?

Reply:
We didn't define our own Logger type. We are using an apex `*log.Logger` directly. Given that we have a package level Logger variable, a package level function to set the log level makes sense to me. Happy to discuss.